crates/turbo-tasks-memory/src/memory_backend.rs

```diff
@@ -591,7 +684,7 @@ pub(crate) enum Job {
     ScheduleWhenDirtyFromScope(AutoSet<TaskId>),
     /// Add tasks from a scope. Scheduled by `run_add_from_scope_queue` to
     /// split off work.
-    AddToScopeQueue(VecDeque<(TaskId, usize)>, TaskScopeId, bool),
+    AddToScopeQueue(VecDeque<TaskId>, TaskScopeId, usize),
```

Comment:
Turn this into a struct variant. It was already unclear what `bool` represented.